src/components/services-view/service-view.classes.ts

```diff
@@ -0,0 +1,181 @@
+import {allCategories} from '../../constants';
+import * as _ from 'lodash';
+
+interface IServiceItem {
+    icon: string;
+    name: string;
+    category: string;
+    subCategory: string;
+    description: string;
+    longDescription: string;
+    data: any;
+}
+
+export class CatalogServiceItem implements IServiceItem {
+    public icon: string;
+    public name: string;
+    public category: string;
+    public subCategory: string;
+    public description: string;
+    public longDescription: string;
+    public data: any;
+
+    constructor (catalogService: any) {
+        this.data = catalogService;
+        this.icon = this.getIcon();
+        this.name = this.getName();
+        this.description = this.getDescription();
+        this.longDescription = this.getLongDescription();
+        this.subCategory = this.getSubCategory();
+        this.category = this.getCategory();
```

Comment:
Ugh, that would require a change to a lot of code in the service view.  How would an item/object be configured to be in multiple categories?  Would it have multiple 'tags' which equal multiple sub-categories?  I was afraid I didn't fully understand the categorization process yet :-(